spec/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tom', posts_counter: 3) }
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+  it 'posts_counter should be an integer greater than or equal to zero' do
```

Comment:
## * Kindly Add unit specs for all of your models' methods as unit tests for them are missing ( for example you have the method `recent_posts` ` update_user_posts_counter` which needs to be tested ) 
![image](https://github.com/torobucci/blog_app/assets/52497246/8955219d-0c4f-41dc-bf1a-ed6ade4df6db)